org.bridgedb.tools.qc/src/org/bridgedb/tools/void/VoIDTool.java

```diff
@@ -0,0 +1,153 @@
+package org.bridgedb.tools.qc;
```

Comment:
Here you need to add a header, showing who wrote it and what the license is. See other BridgeDb code for examples.